includes/Core/Util/Migration_1_128_0.php

```diff
@@ -0,0 +1,187 @@
+<?php
+/**
+ * Migration for 1.128.0
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Modules\Modules;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Modules\Analytics_4\Settings as Analytics_Settings;
+use Google\Site_Kit\Modules\Ads;
+use Google\Site_Kit\Modules\Ads\Settings as Ads_Settings;
+
+/**
+ * Class Migration_1_128_0
+ *
+ * @since 1.128.0
+ * @access private
+ * @ignore
+ */
+class Migration_1_128_0 {
+	/**
+	 * Target DB version.
+	 */
+	const DB_VERSION = '1.128.0';
+
+	/**
+	 * DB version option name.
+	 */
+	const DB_VERSION_OPTION = 'googlesitekit_db_version';
+
+	/**
+	 * Analytics options name.
+	 */
+	const ANALYTICS_OPTIONS = 'googlesitekit_analytics-4_settings';
+
+	/**
+	 * Ads options name.
+	 */
+	const ADS_OPTIONS = 'googlesitekit_ads_settings';
+
+	/**
+	 * Context instance.
+	 *
+	 * @since 1.128.0
```

Comment:
Usually we use `n.e.x.t` in properties/methods comment header, as it will be automatically replaced during release process, and in case this does not make it to the targeted release it eliminates the need to search and replace all instances, so only class would need name changing. But I guess it is ok in this context.